pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -623,37 +638,10 @@ public void failed(ApiService.State from, Throwable failure) {
         consumer.accept(newSubscriber);
     }
 
-    private static void respondToGAEStartHook(String envPort) {
-        var receivedPing = new AtomicBoolean(false);
-
+    private static void startupSpark(String envPort) {
         Spark.port(Integer.parseInt(envPort));
-        Spark.get(RouteConstants.GAE.START_ENDPOINT, (request, response) -> {
-            receivedPing.set(true);
-            response.status(200);
-            return "";
-        });
         Spark.awaitInitialization();
-        log.info("Started HTTP server on port {} and waiting for ping from GAE", envPort);
```

Comment:
Removed complex wait and locking, and renamed method to registerAppEngineCallbacks so we respond to start/stop calls sooner.